sections/talks/torinohacknight.pug

```diff
@@ -285,13 +285,13 @@ html
                       | feedReactions
                     li
                       | imageTag(?)
- 
+
               section
                 h4 and now?
                 ul.left
                   li we need Researchers and Developers of new visualisations!
                   li
-                    | Grants and donation with the help of 
```

Comment:
@gibix every time there is an **a** element is needed a space before and after, otherwise will looks like:
![image](https://cloud.githubusercontent.com/assets/89555/25002006/5db31c86-2048-11e7-90ce-0ebd69ef80a6.png)